open_city_profile/tests/snapshots/snap_test_graphql_api_schema.py

```diff
@@ -182,18 +182,25 @@
 }
 
 type DeleteMyProfileMutationPayload {
+  results: [DeletionResultNode]
   clientMutationId: String
 }
 
-input DeleteMyServiceDataMutationInput {
+input DeleteMyServiceDataInput {
   authorizationCode: String!
   serviceName: String!
   dryRun: Boolean
-  clientMutationId: String
 }
 
-type DeleteMyServiceDataMutationPayload {
-  clientMutationId: String
+type DeleteMyServiceDataMutation {
```

Comment:
The name could still be `DeleteMyServiceDataMutationPayload` as before. That's the naming convention used in all other mutation payload types too.